nautobot/ui/src/components/Layout.js

```diff
@@ -39,15 +95,70 @@ export default function Layout({ children }) {
                         to="/"
                     />
                 </Heading>
-                {isLoggedIn && <SidebarNav />}
-                <Button onClick={legacyUI} variant="link" color="gray-1">
-                    Return to Legacy UI
-                </Button>
+                <Box flex="1">{isLoggedIn && <SidebarNav />}</Box>
+                <Box borderTop="2px" borderColor="#6a6969">
+                    <Button onClick={onOpen} variant="link" color="gray-1">
+                        Leave a Feedback
+                    </Button>
+                    <Button onClick={legacyUI} variant="link" color="gray-1">
+                        Return to Legacy UI
+                    </Button>
+                </Box>
             </Sidebar>
 
             <Box flex="1" height="full">
                 {children}
             </Box>
+
+            <Modal isOpen={isOpen} onClose={onClose} id="22">
+                <ModalOverlay
+                    bg="blackAlpha.300"
+                    backdropFilter="blur(10px)"
+                    zIndex="5"
+                />
+                <ModalContent>
+                    <form method="post" onSubmit={handleLeaveFeedbackSubmit}>
+                        <ModalHeader marginTop="5" marginLeft="5">
+                            Leave a Feedback
+                        </ModalHeader>
+                        <ModalCloseButton />
+                        <ModalBody pb={6}>
+                            <FormControl>
+                                <FormLabel>Email</FormLabel>
+                                <Input
+                                    type="email"
+                                    name="email"
+                                    onChange={changeFeedbackInputValue}
+                                />
+                            </FormControl>
+
+                            <FormControl mt={4}>
+                                <FormLabel>Description</FormLabel>
+                                <Textarea
+                                    border="1px"
+                                    borderColor="gray.300"
+                                    borderRadius="lg"
+                                    width="full"
+                                    padding="2.5px"
+                                    paddingLeft="8px"
+                                    required
+                                    name="description"
+                                    onChange={changeFeedbackInputValue}
+                                    size="lg"
+                                    rows={5}
+                                />
+                            </FormControl>
```

Comment:
Would be good to indicate (as I understand the requirements) that email is optional but description is required?